src/resources/template.html

```diff
@@ -24,5 +24,6 @@
 		<%= htmlWebpackPlugin.options.ssr({
 			url: '/'
 		}) %>
+		<script>"undefined"==typeof fetch&&document.write('<script src="<%= htmlWebpackPlugin.files.publicPath + "polyfills.js" %>"><\/script>')</script>
```

Comment:
Sounds great! I'll change it to `window.fetch==void 0&&document.write(...)` (5 less characters & html webpack plugin doesn't minify inline script 😢 )